src/sidebar/components/test/AnnotationView-test.tsx

```diff
@@ -1,26 +1,43 @@
 import { mockImportedComponents, waitFor } from '@hypothesis/frontend-testing';
 import { mount } from 'enzyme';
+import sinon from 'sinon';
+import type { SinonStub } from 'sinon';
 
-import AnnotationView, { $imports } from '../AnnotationView';
+import { getImports } from '../../../test-util/mockable-imports';
+import type { LoadAnnotationsService } from '../../services/load-annotations';
+import type { SidebarStore } from '../../store';
+import * as AnnotationViewExports from '../AnnotationView';
+import type { AnnotationViewProps } from '../AnnotationView';
 
 describe('AnnotationView', () => {
-  let fakeStore;
-  let fakeOnLogin;
+  const AnnotationView = AnnotationViewExports.default;
+  const $imports = getImports(AnnotationViewExports);
+
+  let highlightAnnotations: SinonStub;
+  let setExpanded: SinonStub;
+  let profile: SinonStub;
+  let fakeStore: SidebarStore;
+  let fakeOnLogin: SinonStub;
   let fakeUseRootThread;
-  let fakeLoadAnnotationsService;
+  let loadThread: SinonStub;
+  let fakeLoadAnnotationsService: LoadAnnotationsService;
 
   beforeEach(() => {
+    highlightAnnotations = sinon.stub();
+    setExpanded = sinon.stub();
+    profile = sinon.stub().returns({ userid: null });
     fakeStore = {
       clearAnnotations: sinon.stub(),
-      highlightAnnotations: sinon.stub(),
+      highlightAnnotations,
       routeParams: sinon.stub().returns({ id: 'test_annotation_id' }),
-      profile: sinon.stub().returns({ userid: null }),
-      setExpanded: sinon.stub(),
-    };
```

Comment:
I would prefer to retain the existing structure where we access stubs as properties of a fake object. Figuring out what a stub refers to is harder without the context of the mock name.

Reply:
Yep, good point.

I'm even seeing more problems that didn't affect these particular tests, but would affect others. For example, when doing this:

```ts
sinon.stub(console, ''warn'');
// ...
console.warn.restore();
```

I'm sure TypeScript would complain `restore` is not a known method of `console.warn`.